lxd/daemon.go

```diff
@@ -527,6 +527,14 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			ctx = context.WithValue(ctx, "protocol", protocol)
 			ctx = context.WithValue(ctx, "access", userAccess)
 
+			// Add forwarded requestor data.
+			if protocol == "cluster" {
+				// Add authentication/authorization context data.
+				ctx = context.WithValue(ctx, "forwarded_address", r.Header.Get("X-LXD-cluster-address"))
```

Comment:
@stgraber please can we have constants defined for these so we can easily see where they are stored and used?

Reply:
And it probably makes sense to have constants for these to:

```go
			ctx := context.WithValue(r.Context(), "username", username)
			ctx = context.WithValue(ctx, "protocol", protocol)
			ctx = context.WithValue(ctx, "access", userAccess)
```